tasks/pkg-redhat.yml

```diff
@@ -64,12 +64,13 @@
         dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
         mode: '600'
         checksum: "sha256:{{ datadog_yum_gpgkey_e09422b3_sha256sum }}"
+      when: agent_datadog_minor is defined and agent_datadog_minor | int < 36
 
     - name: Import new RPM key (Expires in 2022)
       rpm_key:
         key: /tmp/DATADOG_RPM_KEY_E09422B3.public
         state: present
-      when: not ansible_check_mode
+      when: not ansible_check_mode and agent_datadog_minor is defined and agent_datadog_minor | int < 36
```

Comment:
:question: Isn't `agent_datadog_minor` always defined? Or if it can be undefined, what's the expected context?

Or to ask it another way, this appears to invert the logic from the install script (never install the key when minor version isn't set), is that the expected logic?

Reply:
`agent_datadog_minor` can be undefined, meaning install the latest version

Reply:
I might have not been awake enough when I initially reviewed, the logic appears the same :thinking: